### ISSUE TITLE
few things about 1.15a

### ISSUE THREAD

**Issue by Tsin** [DDRace/teeworlds#208]
1. there is only english language in 1.15a build.
2. why there is no icon for linux in build (.png file would do fine)
3. new entities arent good for mappers (why cant we use old):

a) hook wall tile is barely visible or non.
b) freeze tile is purple not blue and too detailed (gives a nice headache).
c) red undeepfreeze was more intuitive.
d) start finish tiles dont have text (why?).
e) solo tiles have now changed colors (maybe its good dont know yet will ask mappers but also no "solo" text" .
g) hammer on hammer off both have that red circle that you cant do it .
h) pink no hook tile dont have that circle .
i) checkpoint in tele layer have no diffrence from normal checkpoints  from game layer (thats bad) .

yes they do look nice but cmon they should be made for mappers not to fucken look nice and give a headache
please lets discuss this problem couse it must be done good for mappers not to look good.

if not then i will have to post old entites on ddmax website  maybe even ddmax will make its own totaly less detailed entities.
maybe we will use even those from 0.5 ddrace (freeze was rly good for eyes).

if this things will be done please make official linux windows and mac build and
i will test everything and ask other mappers to test new build if we wont find anything that is major to fix after lets say 1 week then we should remove all old builds from ddrace.info

**Comment by ghost** [comment]
In linux the executeables themself don't have icons

**Comment by Tsin** [comment]
yep  thats not rly a problem (also i dont rly know linux architecture so i dont know if executables can have icons like on windows)

but what i mean the problem is that tere is no icon in build at all so if i make a launcher i need to go on github search for icon then download it then put it somwhere and then i can use it (ah also i need to convert it to .png couse linux launcher dont want to read .ico files =/)

**Comment by ghost** [comment]
rename the .ico file to .png

**Comment by Tsin** [comment]
haha dont work on linux mint 14 (fresh install) :P

**Comment by ghost** [comment]
works fine for me :>

edit: well, as it seems like you already have the .png file, you could do a pull request
